types/wordpress__data/package.json

```
@@ -1,6 +1,6 @@
 {
     "private": true,
     "dependencies": {
-        "redux": "^4.0.1"
+        "redux": "^4.1.2"
```

[6.0 looks like it should be `^4.1.0`:](https://unpkg.com/browse/@wordpress/data@6.0.0/package.json)

```suggestion
        "redux": "^4.1.0"
```

---

Thanks, you are correct